resources/keycloak/custom-user-provider/pom.xml

```
@@ -0,0 +1,82 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.cvmanager</groupId>
+    <artifactId>keycloak-custom-providers</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>keycloak-custom-providers</name>
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.1.1</version>
+        <relativePath />
+    </parent>
+
+    <dependencies>
+    
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-services</artifactId>
+            <version>${keycloak.version}</version>
+            <exclusions>
+            	<exclusion>
+            		<groupId>org.jboss</groupId>
+            		<artifactId>jandex</artifactId>
+            	</exclusion>
+            </exclusions>
+        </dependency>
+        
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+            <version>${slf4j.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>2.12.3</version> <!-- replace with the latest version -->
```

```suggestion
            <version>2.13.4.2</version>
```
The declared version has multiple vulnerabilities. If possible, we should use the most up-to-date version for this library to protect from said vulnerabilities. One vulnerability is https://osv.dev/vulnerability/GHSA-3x8x-79m2-3w2w

---

Are you using an extension to automatically detect these vulnerabilities?